src/api/apollo/mutations.ts

```diff
@@ -326,7 +326,7 @@ export const CREATE_SCRIPT_EXECUTION = gql`
 `;
 
 export const DELETE_PROJECT = gql`
-  mutation DeleteProject($projectId: UUID) {
+  mutation DeleteProject($projectId: UUID!) {
```

Comment:
what is this doing?

Reply:
it is conforming to the graphql schema, for some reason projectId is optional parameter `!` 
the generator was not happy so I had to add the `!`